development/build/manifest.js

```diff
@@ -37,27 +33,11 @@ function createManifestTasks({ browserPlatforms }) {
 
   // dev: remove bg-libs, add chromereload, add perms
```

Comment:
Nit: Comment could use an update

```suggestion
  // dev: add perms
```